omc3/scripts/kmod_lumi_imbalance.py

```diff
@@ -187,11 +190,13 @@ def get_lumi_imbalance_df(**kwargs) -> tfs.TfsDataFrame:
     Args:
         ipA (tfs.TfsDataFrame): a `TfsDataFrame` with the averaged results from a kmod analysis, for IP_A.
         ipB (tfs.TfsDataFrame): a `TfsDataFrame` with the averaged results from a kmod analysis, for IP_B.
-        (Actually, any name that ends with an integer is fine.)
+        (Actually, any name is fine.)
```

Comment:
You mean for the provided args, right?

Reply:
Feels like lines 131-132 expect some form of nomenclature

Reply:
I want to say that any key for the `kwargs` is fine, just by convention it should be something like "ipA" "ipB" but it doesn't really matter for the function itself. I think the part with the "ends with an integer" was a remainder of some old code, where I only used the  IP number. As I do kind of now in l.132/l.133, for the "ip" labels in the name. But that is a different function and also does not need to be an integer. IPH or IPX would work as well, I am only hoping that the last letter of the name is the main identifier ^^